internal/access_provider/sync.go

```diff
@@ -49,21 +48,16 @@ func (s *DataAccessSync) StartSyncAndQueueJob(client plugin.PluginClient) (job.J
 		defer os.RemoveAll(targetFile)
 	}
 
-	config := data_access.AccessSyncConfig{
-		BaseTargetConfig: *s.TargetConfig,
-	}
-
-	lastUpdated := accessLastCalculated[s.TargetConfig.DataSourceId]
-
 	s.TargetConfig.Logger.Info("Fetching access providers for this data source from Raito")
 	s.StatusUpdater(job.DataRetrieve)
-	dar, err := data_access.RetrieveDataAccessListForDataSource(&config, lastUpdated)
+	daExporter := NewAccessProviderExporter(&AccessProviderExporterConfig{BaseTargetConfig: *s.TargetConfig}, s.StatusUpdater)
+
+	_, dar, err := daExporter.TriggerExport(s.JobId)
 
 	if err != nil {
 		return job.Failed, "", err
 	}
 
-	err = s.updateLastCalculated(dar)
```

Comment:
Why are we removing this? Does this happen somewhere else? 
There is stil a task in my to do to check if this is still valid.

Reply:
that was an accidental removal